django/pyrog/migrations/0005_input_static_value_nullable.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.8 on 2021-06-22 16:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("pyrog", "0004_drop_template"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="input",
+            name="static_value",
+            field=models.TextField(blank=True, default="", null=True),
```

Comment:
what happens to static_value that were previously equal to an empty string ? I think that they should become null, what do you think ?